contracts/sol6/KyberStorage.sol

```diff
@@ -66,10 +67,28 @@ contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers, Utils5 {
         onlyAdmin();
         require(_kyberNetwork != IKyberNetwork(0), "network 0");
         emit KyberNetworkUpdated(_kyberNetwork);
-        previousNetworks.push(kyberNetwork);
+        networkHistory.saveContract(address(_kyberNetwork));
         kyberNetwork = _kyberNetwork;
     }
 
+    function setHistoricalContracts(
```

Comment:
seems to me this better be called init

also its a one time init.
right?
So better comment that.

Reply:
Yes right. In that case, will shift to constructor.